lib/portlayer/storage/nfs/volume.go

```diff
@@ -202,6 +207,11 @@ func (v *VolumeStore) VolumesList(op trace.Operation) ([]*storage.Volume, error)
 	return volumes, nil
 }
 
+// ReadArchive creates and returns a tar archive containing data found between child and parent nfs layers
+func (v *VolumeStore) ReadArchive(op trace.Operation, store *url.URL, child, parent string, pathspec map[string]string, data bool) (io.ReadCloser, error) {
+	return nil, fmt.Errorf("vSphere Integrated Containers does not yet implement ReadArchive for nfs volumes")
```

Comment:
Should be somewhat trivial once all this stuff is in. I was just taking it one step at a time.